v1/topdown/numbers.go

```diff
@@ -98,8 +98,12 @@ func generateCheapRange(operands []*ast.Term, iter func(*ast.Term) error) error
 	step := 1
 
 	if len(operands) > 2 {
```

Comment:
Can this ever happen? Looks like the `numbers.range_step` built-in, that would have the `step` arg, doesn't call to here ..

Reply:
That looks like a mistake, as it should! I'll look into fixing that.

Reply:
I think something like `numbers.range(1, 2, arr)` would still be valid though?